doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+[id='migration-tool-niogit-7.4-proc_{context}']
+
+= Using the migration tool on a `.niogit` directory
+
+The `.niogit` directory of your {CENTRAL} distribution contains all {CENTRAL} data. You can use the {CENTRAL} migration tool on your `.niogit` directory to prepare project data and system configurations for migration to {PRODUCT} 7.5.
+
+
+.Prerequisites
+* {PRODUCT} 7.5 is installed.
+* {PRODUCT} 7.4 in installed and {CENTRAL} is not running on {PRODUCT} 7.4.
+* Java 8 is installed and is available on the path where the project migration tool will be installed.
```

Comment:
JVM? JDK?

Reply:
Tomas, I changed to JDK 8. Please confirm.

Reply:
OK to me.